QuizApp/QuizApp/Login/LoginViewController.swift

```diff
@@ -0,0 +1,111 @@
+import UIKit
+import SnapKit
+
+class LoginViewController: UIViewController {
+
+    var gradientView: GradientView!
+    var mainView: UIView!
+
+    var scrollView: UIScrollView!
+    var contentView: UIView!
+
+    var label: UILabel!
+
+    var stackView: UIStackView!
+
+    var emailView: EmailView!
+    var passwordView: PasswordView!
+    var loginButton: LoginButton!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let tapGestureBackground = UITapGestureRecognizer(target: self, action: #selector(self.backgroundTapped(_:)))
+        self.view.addGestureRecognizer(tapGestureBackground)
```

Comment:
self is not needed. Also, it is better to group actions in functions. So, could you create a `private func addActions()`?